.github/workflows/release.yml

```diff
@@ -23,3 +25,16 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH }}
           NPM_CONFIG_PROVENANCE: true # Enable provenance
 
+  release:
+    runs-on: ubuntu-latest
+    needs: publish
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v4
+      - run: pnpm dlx changelogithub
```

Comment:
Have you tried using that package?
Do you know how much of a quality difference there is from the GitHub auto-generated releases?

Just curious.

Reply:
https://github.com/wrtnlabs/eslint-config
Check it out